multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt

```diff
@@ -88,6 +88,7 @@ class MangaThemesiaGenerator : ThemeSourceGenerator {
         SingleLang("Phoenix Fansub", "https://phoenixfansub.com", "es", overrideVersionCode = 2),
         SingleLang("Pi Scans", "https://piscans.com", "id"),
         SingleLang("PMScans", "https://rackusreads.com", "en", overrideVersionCode = 3),
+        SingleLang("Raiki Scan", "https://raikiscan.com/", "es", className = "RaikiScan"),
```

Comment:
Remove the trailing slash here and setting the `className` isn't needed as it will be automatically generated with the same value you have set.

```suggestion
        SingleLang("Raiki Scan", "https://raikiscan.com", "es"),
```